pubspec.yaml

```diff
@@ -15,41 +15,41 @@ dependencies:
     sdk: flutter
 
   capyscript:
-#    path: ../../StudioProjects/capyscript
-    git: https://github.com/Sayuri128/capyscript
+    path: ../../StudioProjects/capyscript
+#    git: https://github.com/Sayuri128/capyscript
```

Comment:
use local deps only druing development. make sure you change it back to git when making a PR